src/v2/spec.yaml

```diff
@@ -6073,7 +6156,7 @@ paths:
     get:
       tags:
         - Notifications
-      summary: Fetch notifications for a user
+      summary: For a user
```

Comment:
`By user`?

Reply:
it's a notification so should be for, right?